### ISSUE TITLE
CapabilityStatement needs updating

### ISSUE THREAD

**Issue by daniel-thomson** [HL7NZ/hpi#7]
The CapabilityStatement listed in the IG needs some updates (and then we need to return the same thing from the /metadata endpoint - currently the one returned is very bare bones and missing lots of things!):

1. Publisher and Contact should be updated to the Ministry of  Health.
2. Endpoints need updating to https://api.hip.digital.health.nz/fhir/hpi/v1/ (also, should the UAT CapabilityStatement return the UAT address or prod? I'm guessing UAT...)
3. Supported profiles need updating to the proper URLs (currently has HA ones)
4. DHB SearchParameter URI needs updating.

**Comment by davidhay25** [comment]
Think these have all been done. Do you want to confirm?

**Comment by daniel-thomson** [comment]
@pkjordan  Yes - sorry if that was unclear. I was saying it needs changing from the URL including healthalliance to the correct canonical URL for the profile

**Comment by davidhay25** [comment]
That's an error - thought I have fixed it - will go check...

The question I had though, is whether to use the HL7NZ domain - eg
http://hl7.org.nz/fhir/StructureDefinition/hpi-location

or some other one. And yes, resolvability is recommended...

cheers
